impeller/aiks/aiks_context.h

```diff
@@ -31,6 +31,7 @@ class AiksContext {
   std::unique_ptr<ContentContext> content_context_;
   bool is_valid_ = false;
 
+  friend Picture;
```

Comment:
If the caller needs the impeller::Context from the AiksContext, you could just add a getter (completely reasonable IMO) instead of adding a friend.

Reply:
Done.